Classes/Query/Modifier/Faceting.php

```diff
@@ -101,7 +101,6 @@ public function modifyQuery(Query $query)
      */
     protected function buildFacetingParameters()
     {
-        $facetParameters = array();
```

Comment:
This initializes the variable and should not be removed. If at all it should be moved to before line 109